test/models/MdgSub09ResponseSpec.scala

```diff
@@ -19,35 +19,42 @@ package models
 import models.responses.MdgSub09Response
 import play.api.libs.json.{JsValue, Json}
 import utils.SpecBase
+import utils.Utils.emptyString
 
 class MdgSub09ResponseSpec extends SpecBase {
 
-  val EORI1 = "testEori"
+  val eori1 = "testEori"
 
   "The DataModel" should {
+
     "parse message with email and a timestamp" in {
-      val sub09Response = Sub09Response.withEmailAndTimestamp(EORI1)
+      val sub09Response = Sub09Response.withEmailAndTimestamp(eori1)
+
       val result = MdgSub09Response.sub09Reads.reads(sub09Response).get
+
       result.verifiedTimestamp.nonEmpty mustBe true
       result.emailAddress.get mustBe "email@email.com"
     }
 
     "parse message with email and no timestamp" in {
-      val sub09Response = Sub09Response.withEmailNoTimestamp(EORI1)
+      val sub09Response = Sub09Response.withEmailNoTimestamp(eori1)
+
       val result = MdgSub09Response.sub09Reads.reads(sub09Response).get
+
       result mustBe MdgSub09Response(Some("email@email.com"), None)
     }
 
     "parse message no email and no timestamp" in {
-      val sub09Response = Sub09Response.noEmailNoTimestamp(EORI1)
+      val sub09Response = Sub09Response.noEmailNoTimestamp(eori1)
+
       val result = MdgSub09Response.sub09Reads.reads(sub09Response).get
+
       result mustBe MdgSub09Response(None, None)
     }
   }
 
```

Comment:
I think you can delete this space

Reply:
Adam, i have given this line break to differentiate between val and the assertion